src/engine/Drawing/Animation.ts

```diff
@@ -223,7 +223,7 @@ module ex {
          }  
       }
 
-      private _setAnchor(point: Point) {
+      private _setAnchor(point: Vector) {
          //if (!this.anchor.equals(point)) {
```

Comment:
Remove these comments if they don't apply anymore otherwise specify why it's commented